backend/models.py

```diff
@@ -93,18 +97,18 @@ class RecruitmentResult(models.Model):
         ('-', 'Rejected'),
         ('$', 'Signed')
     )
-    upload_request = models.ForeignKey(UploadRequest, on_delete=CASCADE,
+    upload_request = models.ForeignKey(UploadRequest, on_delete=models.CASCADE,
                                        null=True, blank=True)
     student = models.ForeignKey(Candidate, on_delete=models.CASCADE)
     recruitment = models.ForeignKey(Recruitment, on_delete=models.CASCADE)
     points = models.FloatField()
-    result = models.CharField(choices=POSSIBLE_RESULT, max_length=10)
```

Comment:
Wydaje mi się, że lepiej zostawić to pole jako choices, a nie jako stringa, bo łatwiej będzie wyszukiwać/będzie to mniej podatne na błędy.

Reply:
zgadzam się

Reply:
OK, akurat nie ma to znaczenia w kontekście tego co jest w bazie, tak długo jak wiemy co będzie dodawane, ale zmienie to - wydawało mi się, że tak będzie łatwiej i będzie się rzadziej wywalać przy dodawaniu
Zmienię to oczywiście